kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/rocksdb/RocksDBStateStoreConfigSetter.java

```diff
@@ -0,0 +1,98 @@
+package org.hypertrace.core.kafkastreams.framework.rocksdb;
+
+import java.util.Map;
+import org.apache.kafka.streams.state.RocksDBConfigSetter;
+import org.rocksdb.BlockBasedTableConfig;
+import org.rocksdb.BloomFilter;
+import org.rocksdb.CompactionStyle;
+import org.rocksdb.CompressionType;
+import org.rocksdb.Filter;
+import org.rocksdb.InfoLogLevel;
+import org.rocksdb.Options;
+
+public class RocksDBStateStoreConfigSetter implements RocksDBConfigSetter {
```

Comment:
Should we set this as part of the base config?

Reply:
Ok let us do it in the next PR after testing it. Not sure what happens if rocksdb config doesn't exist ?